parser/convert.cpp

```diff
@@ -0,0 +1,50 @@
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+  return 0;
+}
+
+int ConvertExcelDateToBasicDate(excelDate timestamp, basicDate* converted) {
```

Comment:
This actually won’t work for January and February 1900.

Reply:
Wow, nice catch!